api-server/package.json

```diff
@@ -72,22 +65,17 @@
     "@babel/plugin-proposal-optional-chaining": "^7.11.0",
     "@babel/preset-env": "^7.12.1",
     "@babel/register": "^7.12.1",
-    "adler32": "~0.1.7",
     "babel-core": "^7.0.0-bridge.0",
     "babel-jest": "^24.9.0",
     "babel-plugin-transform-function-bind": "^6.22.0",
     "babel-plugin-transform-imports": "^1.5.1",
     "cross-env": "^6.0.3",
     "jest": "^26.6.1",
-    "joi": "^13.7.0",
-    "joi-objectid": "^2.0.0",
     "loopback-component-explorer": "^6.4.0",
     "nodemon": "^2.0.6",
     "passport-mock-strategy": "^2.0.0",
     "pm2": "^4.5.0",
```

Comment:
On the VM the demo-projects are stored, pm2 is installed globally and not at the repo level.  Could we remove this if it is installed globally?

Reply:
Good question. I wasn't sure about pm2, since it manages the api, but isn't part of it.

I think you're right and it should probably go.  @raisedadead could you confirm?

Reply:
It can go.